app/controllers/users_controller.rb

```diff
@@ -6,7 +6,7 @@ class UsersController < ApplicationController
   def edit
     unless current_user
       skip_authorization
-      return redirect_to "/enter"
+      return redirect_to new_user_registration_path_path
```

Comment:
can we also change the alias into `new_user_registration_path` in routes?

Reply:
`new_user_registration_path` has already been declared by devise. What else do you reckon? :)

Reply:
okay, keep it then